rotkehlchen/tests/api/test_assets.py

```diff
@@ -445,11 +445,15 @@ def test_get_assets_mappings(rotkehlchen_api_server):
         json={'identifiers': queried_assets},
     )
     result = assert_proper_response_with_result(response)
-    assert len(result) == 3
-    for identifier in result.keys():
+    assert len(result) == len(queried_assets)
+    for identifier, details in result.items():
         assert identifier in queried_assets
+        if identifier == queried_assets[-1]:
```

Comment:
I believe it would be better to move the identifier to a variable and then use it in the tuple and here. Then it is easier to quickly read what asset is being targeted and you prevent errors if the tuple is changed in the future